src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.event;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+import seedu.address.model.person.Name;
+
+public class Event {
```

Comment:
@limweiliang Please confirm - shouldn't an `Event` be tied to `Person` object(s)?